frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -511,134 +508,7 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
     selectedUpdateChannel,
   ).filter((item) => !isInternalObject(internalObjects, item.name));
 
-  return (
```

Comment:
This appears way more changed than it actually is.  I simple made the original output a variable so I could show either the install page or the installing view. Sorry for the mess.